rest-api/recipe/routes/recipe-routes.js

```diff
@@ -16,4 +19,8 @@ router.get('/', getRecipes);
 router.get('/:id', validateId, getRecipeById);
 router.post('/', validateToken, requiredFields, addRecipe);
 
+//sam
+router.put('/recipe/:id', validateId, requiredFields, editRecipe)
+router.put('/tags/:id', validateId, requiredFields, editTag)
```

Comment:
What is this the `id` of? Because if it's the recipe, the path should be: `/:id/tags`. But then it's ambiguous as to which tag. So maybe you need something like: `/:id/tags/:tag_id`?

Reply:
Yh!!! you are right. In this case, I'm thinking `/:id/tags` makes sense .
and i'm getting rid of all the middlewares. We'll add them as and when they are needed

Reply:
Alright!